slam-libraries/viam-cartographer/src/slam_service/slam_service_test.cc

```diff
@@ -69,45 +72,308 @@ BOOST_AUTO_TEST_CASE(OverwriteMapBuilderParameters_set_values_mapping) {
     slamService.occupied_space_weight = 9990.09;
     slamService.translation_weight = 9989.89;
     slamService.rotation_weight = 9988.88;
+
+    BOOST_TEST(slamService.GetActionMode() == SLAMServiceActionMode::MAPPING);
```

Comment:
`map_rate_sec` is irrelevant in this specific test since it does not change the outcome and is not what I'm testing for. I am checking for the state to be in "MAPPING" though cause that's the prerequisite for the test to succeed, and that's what the `action_mode` is expected to be at.

In other tests, I have to set `map_rate_sec` to a specific value to change the `action_mode` state so I can make sure that the behavior is as expected.

Reply:
correct :)